exporters/otlp/otlptrace/README.md

```diff
@@ -0,0 +1,28 @@
+# OpenTelemetry-Go OTLP Span Exporter
+
+[![Go Reference](https://pkg.go.dev/badge/go.opentelemetry.io/otel/exporters/otlp/otlptrace.svg)](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace)
+
+OpenTelemetry Protocol (OTLP) Span Exporter.
```

Comment:
```suggestion
[OpenTelemetry Protocol Exporter](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.5.0/specification/protocol/exporter.md) implementation.
```

Reply:
I can also adopt this when I create otlpmetric README